### PR TITLE
OGS under construction

### PR DESCRIPTION
**Pull request by mistakevin**
includes https://github.com/OpenSourcePolicyCenter/webapp-public/pull/178 to prevent css conflicts

screen shot 2016-03-10 at 9 49 51 am

### CODE REVIEW

**Comment by talumbau** [comment]
@mistakevin, Can they still click through? The desire is to be able to click through and then have a second "under construction" banner of some kind with more text. Please confirm @MattHJensen

**Comment by mistakevin** [comment]
Ah, no, I disabled it altogether. I can re-enable it

File: templates/dynamic/landing.html [hunk 1]
```diff
@@ -65,14 +65,21 @@ <h2 class="sr-only">TaxBrain</h2>
...
                     <td>
-                    <a href="/dynamic/ogusa/{{pk}}/?start_year={{start_year}}" class="text-white btn btn-secondary">Overlapping Generations Simulation</a>
+                    <a href="/dynamic/ogusa/{{pk}}/?start_year={{start_year}}" class="text-white btn btn-secondary disabled">Overlapping Generations Simulation</a>
                     </td>
                     <td>
+                      <div class="construction">
+                        <div class="progress progress-striped" style="min-width: 400px; width: 100%">
+                          <div class="progress-bar active progress-bar-custom" role="progressbar" aria-valuenow="45" aria-valuemin="0" aria-valuemax="100" style="width: 100%"></div>
+                        </div>
+                        <small>Under Construction - Coming Soon!</small>
```
Comment (MattHJensen) [inline templates/dynamic/landing.html]: I can stick this, "Under Construction - Coming Soon!" in the "dynamic_olg_blurb" flatblock.

**Comment by MattHJensen** [comment]
> @mistakevin, Can they still click through? The desire is to be able to click through and then have a second "under construction" banner of some kind with more text. Please confirm @MattHJensen

Confirmed.

**Comment by talumbau** [comment]
@mistakevin, I ran this locally. The page looks good. When you click through to the OG-USA input page, there should be another construction bar with the words "Under Construction". The button for "Submit Calculation" should be grayed out or unclickable in some way. Make sense?

**Comment by mistakevin** [comment]
Yep.

**Comment by talumbau** [comment]
@mistakevin this looks good to me. Merging.

**Pull request merged**